### ISSUE TITLE
Time conversion error at the start of the universe

### ISSUE THREAD

**Issue by larsgw** [maxlath/wikidata-sdk#25]
## Error

During routine tests (using `Q1` as a simple id), I got this error, a lot:

```
wikidata-sdk time conversion error: RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at toISOString (/home/larsw/public_html/Citation/node_modules/wikidata-sdk/lib/helpers/helpers.js:28:35)
    at value (/home/larsw/public_html/Citation/node_modules/wikidata-sdk/lib/helpers/helpers.js:19:12)
    at Object.time (/home/larsw/public_html/Citation/node_modules/wikidata-sdk/lib/helpers/parse_claim.js:41:49)
    at module.exports (/home/larsw/public_html/Citation/node_modules/wikidata-sdk/lib/helpers/parse_claim.js:76:32)
    at simplifyClaim (/home/larsw/public_html/Citation/node_modules/wikidata-sdk/lib/helpers/simplify_claims.js:85:17)
    at propClaims.map.claim (/home/larsw/public_html/Citation/node_modules/wikidata-sdk/lib/helpers/simplify_claims.js:33:19)
    at Array.map (<anonymous>)
    at simplifyPropertyClaims (/home/larsw/public_html/Citation/node_modules/wikidata-sdk/lib/helpers/simplify_claims.js:33:6)
    at Object.simplifyClaims (/home/larsw/public_html/Citation/node_modules/wikidata-sdk/lib/helpers/simplify_claims.js:14:24)
```

## Input

To be fair, the time values being parsed *are* pretty long ago, but I don't think the values themselves are invalid:

```
{ time: '-13798000000-00-00T00:00:00Z',
  timezone: 0,
  before: 0,
  after: 0,
  precision: 3,
  calendarmodel: 'http://www.wikidata.org/entity/Q1985727' }
```

## Code

Run on [RunKit](https://runkit.com/embed/3igsu88ql20d).

```js
const wdk = require("wikidata-sdk")
require('isomorphic-fetch')

const data = await fetch(wdk.getEntities('Q1')).then(response => response.json())

wdk.simplify.entity(data.entities.Q1)
```

## Solution

None, it seems to be a limitation of JavaScript `Date`:

```js
const d = new Date()
d.setYear(13798000000) // NaN
d // Invalid date
```

**Comment by maxlath** [comment]
which version of `wikidata-sdk` are you using?

**Comment by larsgw** [comment]
`v5.9.0`. In `v5.2.2` I didn't get the error, but I don't think the conversions are working either: https://runkit.com/larsgw/5adf8b05d3216e0014bde1f6

**Comment by maxlath** [comment]
I tried a work around 8f68827 (published in `v5.9.1`), but it doesn't work for all conversion formats: we can't get `epoch` time for this kind of date. Could you try it to see if it addresses your need?

**Comment by larsgw** [comment]
It works great, thanks!
